docs/components/connectors/out-of-the-box-connectors/embeddings-vector-db.md

```diff
@@ -0,0 +1,147 @@
+          "customProperties": {},
+          "expiresAt": null,
+          "contentType": "text/plain"
+        },
+        "camunda.document.type": "camunda"
+      },
+      "score": 0.6721556,
+      "content": "Camunda is a platform for orchestrating and automating business processes. It helps organizations design, execute, and manage workflows, enabling them to optimize processes and improve efficiency."
+    }
+  ]
+}
+```
+
+## Embedding models
+
+### Amazon Bedrock
+
+The **Vector Database Connector** currently supports only [Amazon Titan V1/V2 models](https://docs.aws.amazon.com/bedrock/latest/userguide/titan-embedding-models.html).
+Please see an [official Amazon documentation](https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-titan-embed-text.html)
+that will explain what are and how to choose request parameters.
+
+The **Vector Database Connector** uses [Langchain4J implementation](https://docs.langchain4j.dev/integrations/embedding-models/amazon-bedrock).
+
+## Vector stores
+
+### Elasticsearch
+
+The **Vector Database Connector** can use Elasticsearch as a vector store. The Elasticsearch version must be 8+.
+
+Enter the following parameters:
+
+- **Base URL**: the Elasticsearch base URL, including protocol, i.e. `https://host:port`.
+- **Username**: for the Elasticsearch user that has read/write access.
+- **Password**: for the Elasticsearch user that has read/write access.
+- **Index name**: name of the index where you wish to store embeddings.
+- - When embedding: if index is not present, the Connector will create a new one.
+- - When retrieving: if the index is absent, the Connector will raise an error.
+
+### Amazon Managed Opensearch
+
+- **Access key** and **Secret key**: enter AWS IAM credential for the user that has read/write access.
+- **Server URL**: is an Amazon Managed Opensearch URL _without_ protocol, i.e. `my-opensearch.aws.com:port`.
+- **Region**: of the Amazon Managed Opensearch instance.
+- **Index name**: name of the index where you wish to store embeddings.
+- - When embedding: if index is not present, the Connector will create a new one.
+- - When retrieving: if the index is absent, the Connector will raise an error.
+
+## Embedding document configuration
+
+### Document source
+
+The **Document source** can be either a **Plain text** or a **Camunda document**.
+
+The **Plain text** can be useful when you deal with small size data that can fit into a text field or a process instance variable.
+Input will be handled as a regular UTF-8 text.
+
+Hint: a FEEL [string conversion function](/components/modeler/feel/builtin-functions/feel-built-in-functions-conversion/#stringfrom) might be useful if you have JSON input.
+
+The **Camunda document** might be useful when you deal with larger documents pipelines that come either from
+[webhook or user tasks](/guides/document-handling). Input documents will be parsed with [Apache Tika](https://tika.apache.org/) parser, hence files
```

Comment:
"will be parsed with Apache Tika, hence"?

Reply:
```suggestion
[webhook or user tasks](/guides/document-handling). Input documents will be parsed with [Apache Tika](https://tika.apache.org/), hence files
```